examples/taint-checking/taint-checking.description

```diff
@@ -1,12 +1,9 @@
-//Ballerina programs are inherently resilient to major security
-//vulnerabilities including SQL injection, path manipulation,
-//file manipulation, unauthorized file access and unvalidated
+//Ballerina programs are secure by design. They are inherently resilient 
+//to major security vulnerabilities such as SQL injection, path manipulation,
+//file manipulation, unauthorized file access, and unvalidated
 //redirect (open redirect).
 //
-//This added protection will make sure Ballerina programs are
-//"secure by design". In order to provide this protection,
```

Comment:
Any reason to drop "secure by design" point? That is very important point IMO.

Reply:
I have added that to the beginning of the comment when making it a bit more concise.